src/rln_contract.ts

```diff
@@ -91,6 +91,13 @@ export class RLNContract {
     const membershipKey = await rlnInstance.generateSeededMembershipKey(
       signature
     );
+
+    return this.registerMemberFromMembershipKey(membershipKey);
+  }
+
+  public async registerMemberFromMembershipKey(
```

Comment:
```suggestion
  public async registerMemberUsingMembershipKey(
```

What about just having the same function handle both scenarios?

Reply:
Renaming to `registerWithSignature` and `registerWithKey`.

Reasoning for it is to allow explicit import of the credentials - exactly what we support in our example.

![image](https://user-images.githubusercontent.com/118575614/233168455-342cea36-5334-4c55-a41b-22226c6d57eb.png)